clients/web/src/views/body/PluginsView.js

```diff
@@ -57,6 +57,7 @@ var PluginsView = View.extend({
     initialize: function (settings) {
         cancelRestRequests('fetch');
         if (settings.all && settings.enabled) {
+            this.cherrypyServer = settings.cherrypyServer || true;
```

Comment:
Won't this construct make it impossible to set `this.cherrypyServer` to false? I usually use a ternary and `_.has` for cases where the default is true.